src/app/home/home.component.ts

```diff
@@ -0,0 +1,16 @@
+import {Component, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.less']
+})
+export class HomeComponent implements OnInit {
+
+  constructor() {
+  }
+
+  ngOnInit(): void {
```

Comment:
Unnecessary hooks and methods should be removed

Reply:
Removed